render/bezier.go

```diff
@@ -0,0 +1,48 @@
+package render
+
+import (
+	"image"
+	"image/color"
+
+	"github.com/oakmound/oak/alg"
+	"github.com/oakmound/oak/alg/intgeom"
+	"github.com/oakmound/oak/shape"
+)
+
+// BezierLine converts a bezier into a line sprite.
+func BezierLine(b shape.Bezier, c color.Color) *Sprite {
+	low := 0.0
+	high := 1.0
+	pts := make([]intgeom.Point, 2)
+	pts[0] = roundToIntPoint(b.Pos(low))
+	pts[1] = roundToIntPoint(b.Pos(high))
+	bezierDraw(b, &pts, low, high, pts[0], pts[1])
+
+	min := pts[0].LesserOf(pts...)
+	max := pts[0].GreaterOf(pts...)
+
+	rgba := image.NewRGBA(image.Rect(0, 0, 1+(max.X-min.X), 1+(max.Y-min.Y)))
+
+	for _, p := range pts {
+		rgba.Set(p.X-min.X, p.Y-min.Y, c)
+	}
+
+	return NewSprite(float64(min.X), float64(min.Y), rgba)
+}
+
+func roundToIntPoint(x, y float64) intgeom.Point {
```

Comment:
There might be some place in a more general place for this helper function. Perhaps something in a sub-package of alg?

Reply:
I think it would go in alg itself, considering that's where `RoundF64` already lives.